File: test/csharp/xlsx.cs
using DocRaptor.Client;
using DocRaptor.Model;
using DocRaptor.Api;
using System;
using System.IO;
using System.Threading;

namespace DocRaptorConsoleExample {
  class DocRaptor {
    static void Main(string[] args) {
      Configuration.Username = "YOUR_API_KEY_HERE";
      // Configuration.Debug = true; // Not supported in Csharp
      DefaultApi docraptor = new DefaultApi();

      Doc doc = new Doc();
      doc.Name = "swagger-csharp.xlsx";
      doc.Test = true;
      doc.DocumentContent = "<html><body><table><tr><td>Swagger C#</td></tr></table></body></html>";
      doc.DocumentType = "xlsx";

      Stream response = docraptor.DocsPost(doc);
    }
  }
}

File: test/node/xlsx.js
var fs = require('fs');
var Q = require("q");

var Docraptor = require("../../clients/node/docraptor.js");
var docraptor = new Docraptor.Test();
docraptor.setToken("YOUR_API_KEY_HERE", "user_credentials", true);

require('request').debug = true;

var request = docraptor.postDocs({
  "doc": {
    "name": "swagger-node.xlsx",
    "document_type": "xlsx",
    "document_content": "<html><body><table><tr><td>Swagger Node</td></tr></table></body></html>",
    "test": true
  }
});

request.done(function(result) {
  res = result;
  console.log(result);

  fs.writeFile('/tmp/node.xlsx', res.response.body, 'binary', function(err){
    if (err) throw err
    console.log("The file /tmp/node.xlsx was saved!");
  });
});
